src/Stuart/Converters/JsonToJob.php

```diff
@@ -3,7 +3,12 @@
 namespace Stuart\Converters;
 
 use Stuart\Job;
+use Stuart\Pricing;
 
+/**
+ * Class JsonToJob
+ * @package Stuart\Converters
```

Comment:
Is this annotation required here considering we already have the `namespace`? Moreover, if you think it's important to have it here, please update every other classes with this annotation to keep a homogenous codebase.

Reply:
Actually, not required here. No use of it. Will remove it